sbnana/SBNAna/NuMINumuXSec/ICARUSNumuXsec_Cuts.h

```diff
@@ -0,0 +1,112 @@
+#ifndef ICARUSNumuXsec_Cuts_h
+#define ICARUSNumuXsec_Cuts_h
+
+#include "sbnana/SBNAna/NuMINumuXSec/ICARUSNumuXsec_Variables.h"
+
+namespace ICARUSNumuXsec{
+
+  //==== NC
+  const Cut cutIsNuMuNC([](const caf::SRSliceProxy* slc) {
+      return ( kIsNuSlice(slc) && slc->truth.isnc && ( slc->truth.pdg == 14 || slc->truth.pdg == -14 ) );
+    });
+
+  const Cut cutTruthNoPiZero([](const caf::SRSliceProxy *slc){
+    return (varTruthNPiZero(slc)==0);
+  });
```

Comment:
For a cut like this you can just write
```suggestion
  const Cut cutTruthNoPiZero = (varTruthNPiZero == 0);
```